docs/quickstart/index.md

```diff
@@ -1,14 +1,53 @@
 ---
 layout: default
-title: Quickstart
+title: QuickStart
 description: The quickstart. This section provides simple instructions for deploying lakeFS so that you start exploring the technology
 nav_order: 5
 has_children: true
 redirect_from: "../quickstart.html"
 has_toc: false
+next: ["Try lakeFS without installing", "./try.html"]
 ---
 
-# Quickstart
+# QuickStart
```

Comment:
It's confusing that this page is similar - but not identical to - the next one ("try w/o installing").
It's also very busy with videos and it can be overwhelming.